### DISCUSSION TITLE
Kindly ask about installing PyG

### DISCUSSION THREAD

**Discussion by ShijieMin** [pyg-team/pytorch_geometric#7566] [Q&A]
I tried to install PyG on my local computer(Windows). But it installing it for hours(I just open my computer overnight) also still has a lot of conflicts with other packages. In this situation, I am not sure if I should reinstall all the conflict packages or there are other solutions.

The reason I am struggling is that I am reinstall the packages before to avoid other conflicts. So it is hard to reinstall all these again and also the installment for PyG takes hours so it is hard to check which package has issues with it.

For installing, I use code: conda install pyg -c pyg
I use Visual Studio Code via Anaconda.

Thank you for the help!

**Comment by wsad1** [comment]
Installation via conda isn't supported for windows. See insatallation instructions [here](https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html). Use `pip` to install instead.
